src/main/kotlin/no/uio/microobject/type/ReasonerMode.kt

```diff
@@ -0,0 +1,5 @@
+package no.uio.microobject.type
```

Comment:
Please move this enum definition to Main.kt, it is related to the settings and needs no own file